### ISSUE TITLE
mWeb - Workspace - Invitation with apostrophe received by SMS is converted to uni message

### ISSUE THREAD

**Issue by lanitochka17** [Expensify/App#51424]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 9.0.53-0
**Reproducible in staging?:** Y
**Reproducible in production?:** Y
**If this was caught on HybridApp, is this reproducible on New Expensify Standalone?:**N/A 
**If this was caught during regression testing, add the test name, ID and link from TestRail:** N/A
**Issue reported by:** Applause - Internal Team

## Action Performed:
1. Open the staging.new.expensify.com website
2. Tap on settings
3. Tap on "Workspaces"
4. Open any workspace
5. Tap on "Members
6. Tap on "Invite Members"
7. Select any user that has a contact number as primary contact method
8. Write an invitation that contains an apostrophe. (Testing´s invitation)
9. Tap on "Invite"
10. Wait until receiving the invitation SMS
11. Verify that the invitation message is correctly displayed in the received SMS

## Expected Result:
The SMS received with the invitation to the workspace should be displayed excatly as the user writed it, even when it has an apostrophe on it

## Actual Result:
Invitation message received by SMS is converted to uni message when it contains an apostrophe

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Standalone
- [ ] Android: HybridApp
- [ ] Android: mWeb Chrome
- [ ] iOS: Standalone
- [ ] iOS: HybridApp
- [ ] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

<details>
  <summary>Add any screenshot/video evidence</summary>

  
</details> 

https://github.com/user-attachments/assets/94029a62-77fa-4903-95a9-08a9a127761d

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)
![Bug6617613_1727457212545!UNI](https://github.com/user-attachments/assets/a9419a4b-2d3b-49ff-b57b-8b88761dc358)

**Comment by lanitochka17** [comment]
@zanyrenney  FYI I haven't added the External label as I wasn't 100% sure about this issue. Please take a look and add the label if you agree it's a bug and can be handled by external contributors

**Comment by zanyrenney** [comment]
Hmm this is a weird one for sure! Quite niche but we should fix.

**Comment by zanyrenney** [comment]
Looking for proposals here.

**Comment by abdulrahuman5196** [comment]
Hi, I was OOO couple of days, checking now

**Comment by abdulrahuman5196** [comment]
Checking now again

**Comment by abdulrahuman5196** [comment]
@NJ-2020 's proposal here https://github.com/Expensify/App/issues/51424#issuecomment-2436714207 looks good and works well.

🎀 👀 🎀
C+ Reviewed

**Comment by abdulrahuman5196** [comment]
@NJ-2020 's I am still able to repro the issue and SMS is being received. Could you try again? Maybe with different phone numbers? Or how you tested during proposal?

**Comment by zanyrenney** [comment]
@abdulrahuman5196 what is your suggestion here ? Thanks!

**Comment by yuwenmemon** [comment]
@abdulrahuman5196 Would it be possible for you to test using @NJ-2020's branch [here](https://github.com/Expensify/App/pull/52684) so we can move this along?

**Comment by abdulrahuman5196** [comment]
> @abdulrahuman5196 Would it be possible for you to test using @NJ-2020's branch [here](https://github.com/Expensify/App/pull/52684) so we can move this along?

Yes. I can. I tested during Proposal phase and it was working fine. @NJ-2020 Could you publish the PR, I will try to test it out at my end

**Comment by zanyrenney** [comment]
Not quite yet @NJ-2020 @abdulrahuman5196 please can you complete the checklist here and ping me when thats done so I can issue payouts. https://github.com/Expensify/App/issues/51424#issuecomment-2560294998

Thanks!

**Comment by zanyrenney** [comment]
I have DM'd @abdulrahuman5196

**Comment by abdulrahuman5196** [comment]
# BugZero Checklist:

- [ ] **[Contributor]** Classify the bug:

<summary>Bug classification</summary>


Source of bug:
  - [ ] 1a. Result of the original design (eg. a case wasn't considered)
  - [ ] 1b. Mistake during implementation
  - [ ] 1c. Backend bug
  - [ ] 1z. Other:

Where bug was reported:
  - [ ] 2a. Reported on production (eg. bug slipped through the normal regression and PR testing process on staging)
  - [ ] 2b. Reported on staging (eg. found during regression or PR testing)
  - [ ] 2d. Reported on a PR
  - [ ] 2z. Other:

Who reported the bug:
  - [ ] 3a. Expensify user
  - [ ] 3b. Expensify employee
  - [ ] 3c. Contributor
  - [ ] 3d. QA
  - [ ] 3z. Other:

- [ ] **[Contributor]** The offending PR has been commented on, pointing out the bug it caused and why, so the author and reviewers can learn from the mistake.

    Link to comment:

- [ ] **[Contributor]** If the regression was CRITICAL (e.g. interrupts a core flow) A discussion in [#expensify-open-source](https://app.slack.com/client/E047TPA624F/C01GTK53T8Q) has been started about whether any other steps should be taken (e.g. updating the PR review checklist) in order to catch this type of bug sooner.

    Link to discussion:

- [ ] **[Contributor]** If it was decided to create a regression test for the bug, please propose the [regression test](https://github.com/Expensify/App/blob/main/contributingGuides/REGRESSION_TEST_BEST_PRACTICES.md) steps using the template below to ensure the same bug will not reach production again.

<summary>Regression Test Proposal Template</summary>


- [ ] **[BugZero Assignee]** Create a GH issue for creating/updating the regression test once above steps have been agreed upon.

    Link to issue:

## Regression Test Proposal
### Precondition:


-

### Test:


1.

Do we agree 👍 or 👎

**Comment by abdulrahuman5196** [comment]
@zanyrenney Sorry for the delay. Added the checklist.
